### PR TITLE
Issue 2414 gui testcases

### PR DESCRIPTION
**Pull request by maryvictol**
This PR add gui testcases for Notify administrators if node pool reached its max size and all nodes in use #2414

### CODE REVIEW

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+**Prerequisites**:
+- Admin user
+- Non-admin user
+
+**Preparations**:
+1. Login as the admin user from the Prerequisites
+2. Open the **Cluster state** page
+3. Click the **HOT NODE POOLS** tab
+4. Click the "**+ Create**" button
+5. Specify follow values
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md:14]: Add specifying of the "Nodes count" value

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+| 7 | At the tool page, hover over the **Run v** button | |
+| 8 | Click the **Custom settings** button in the list | |
+| 9 | Expand the **Exec environment** section | |
+| 10 | Set *Node type* the same as *Instance type* specified for node pool at step 5 of the Preparations | |
+| 11 | Set *Disk* as `15` | | 
+| 12 | Set *Cloud Region* the same as *Region* specified for node pool at step 5 of the Preparations | |
+| 13 | Expand the **Advanced** section | | 
+| 14 | Select **Spot** Price type | |
+| 15 | Launch the run | |
+| 16 | Repeat steps 5-14 | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md]: Fix to `Repeat steps 6-15`

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+| 17 | Open the **Cluster state** page | |
+| 18 | Click the **HOT NODE POOLS** tab | |
+| 19 | Wait until **2/2** nodes is shown in the pool state for pool created at step 8 of the Preparations |
+| 20 | Wait about 1 minutes | | 
+| 21 | Check admin and non-admin users email | Non-admin and admin users from the Prerequisites receive email that contains <li> node pool name <li> node pool maximum size <li> instance type |
+
+**After:**
+- Stop  runs launched at steps 15, 16 from the Preparations
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md]: ~from the Preparations~

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- Open the **HOT NODE POOLS** tab and delete the pool node created at step 8 of the Preparations
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md]: Fix to `... delete the node pool ...`

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+**Preparations**:
+1. Repeat steps 1-9 of the Preparations of [2414_1](2414_1.md) case 2 times with different docker images (*tool1* and *tool2*)
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Select test *tool1* | |
+| 2 | Repeat steps 7-15 of the [2414_1](2414_1.md) case | |
+| 3 | Repeat steps 1-2 | |
+| 4 | Select test *tool2* | |
+| 5 | Repeat step 2 | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md]: For more understandable, fix to `Repeat steps 7-15 of the [2414_1](2414_1.md) case`

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+1. Repeat steps 1-9 of the Preparations of [2414_1](2414_1.md) case 2 times with different docker images (*tool1* and *tool2*)
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Select test *tool1* | |
+| 2 | Repeat steps 7-15 of the [2414_1](2414_1.md) case | |
+| 3 | Repeat steps 1-2 | |
+| 4 | Select test *tool2* | |
+| 5 | Repeat step 2 | | 
+| 6 | Repeat steps 5-6 | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md]: Fix to `Repeat steps 4-5`

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+| 7 | Open the **Cluster state** page | |
+| 8 | Click the **HOT NODE POOLS** tab | |
+| 9 | Wait until **2/2** nodes is shown in the pool state for both pools created at step 1 of the Preparations |
+| 10 | Wait about 1 minutes | | 
+| 11 | Check admin and non-admin users email | Non-admin and admin users from the Prerequisites receive 1 email for both pools that contains follow information for each pool <li> node pool name <li> node pool maximum size <li> instance type |
+
+**After:**
+- Stop  runs launched at steps 2, 3, 5, 6 from the Preparations
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md]: ~from the Preparations~

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+- Open the **HOT NODE POOLS** tab and delete the pool node created at step 1 of the Preparations
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md]: Fix to `... delete the node pools created ...`

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+| 13 | Open the **Cluster state** page | 
+| 14 | Click the **HOT NODE POOLS** tab | **2/2** nodes is shown in the pool state for pool created at step 8 of the Preparations |
+| 15 | Wait about 1 minutes. Check admin and non-admin users email | Non-admin and admin users from the Prerequisites don't receive emails | |
+| 16 | Repeat 2 times steps 1-11 |  |
+| 17 | Open the **Cluster state** page |
+| 18 | Click the **HOT NODE POOLS** tab | **4/4** nodes is shown in the pool state for pool created at step 8 of the Preparations |
+| 19 | Wait about 1 minutes. Check admin and non-admin users email | Non-admin and admin users from the Prerequisites receive emails | |
+
+**After:**
+- Stop  runs launched at steps 11, 12, 16 from the Preparations
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md]: ~from the Preparations~

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+- Open the **HOT NODE POOLS** tab and delete the pool node created at step 8 of the Preparations
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md]: Fix to `... delete the node pool ...`

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+| 6 | Set *Node type* the same as *Instance type* specified for node pool at step 5 of the Preparations | |
+| 7 | Set *Disk* as `15` | | 
+| 8 | Set *Cloud Region* the same as *Region* specified for node pool at step 5 of the Preparations | |
+| 9 | Expand the **Advanced** section | | 
+| 10 | Select **Spot** Price type | |
+| 11 | Launch the run | |
+| 12 | Repeat steps 1-11 |  |
+| 13 | Open the **Cluster state** page | 
+| 14 | Click the **HOT NODE POOLS** tab | **2/2** nodes is shown in the pool state for pool created at step 8 of the Preparations |
+| 15 | Wait about 1 minutes. Check admin and non-admin users email | Non-admin and admin users from the Prerequisites don't receive emails | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md]: Add `... don't receive emails about hot node pools fullness`

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+| 16 | Repeat 2 times steps 1-11 |  |
+| 17 | Open the **Cluster state** page |
+| 18 | Click the **HOT NODE POOLS** tab | **4/4** nodes is shown in the pool state for pool created at step 8 of the Preparations |
+| 19 | Wait about 1 minutes. Check admin and non-admin users email | Non-admin and admin users from the Prerequisites receive emails | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md]: Add `... received emails about hot node pools fullness`

**Review by NShaforostov — APPROVED** [review]
Fix minor notes before merging, please (see comments for details).

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md (hunk 12 above)
Comment (maryvictol) [reply docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md]: Done

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md (hunk 11 above)
Comment (maryvictol) [reply docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md]: Done

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md (hunk 10 above)
Comment (maryvictol) [reply docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md]: Done

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md (hunk 9 above)
Comment (maryvictol) [reply docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_3.md]: Done

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md (hunk 7 above)
Comment (maryvictol) [reply docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md]: Done

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md (hunk 8 above)
Comment (maryvictol) [reply docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md]: Done

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md (hunk 6 above)
Comment (maryvictol) [reply docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md]: Done

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md (hunk 3 above)
Comment (maryvictol) [reply docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md]: Done

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md (hunk 4 above)
Comment (maryvictol) [reply docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_1.md]: Done

File: docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md (hunk 5 above)
Comment (maryvictol) [reply docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md]: Done

**Pull request merged**